Common++/header/PointerVector.h

```diff
@@ -56,7 +56,9 @@ namespace pcpp
 		 * @param[in] other The vector to move from.
 		 */
 		PointerVector(PointerVector&& other) noexcept : m_Vector(std::move(other.m_Vector))
-		{}
+		{
+			other.m_Vector.clear();
```

Comment:
The vector move ctor technically does not have a well defined state after move. The standard just says it should be in a valid state, but it does not guarantee it will always be empty state. It ysually is, but that is implementation defined.

Reply:
I see. thanks for the explanation.